src/game/client/draw/w3dsupplydraw.cpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D Supply Draw Module
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3dsupplydraw.h"
+#include "drawable.h"
+
+void W3DSupplyDrawModuleData::Build_Field_Parse(MultiIniFieldParse &p)
+{
+    // clang-format off
+    static const FieldParse dataFieldParse[] = {
+        { "SupplyBonePrefix", &INI::Parse_AsciiString, nullptr, offsetof(W3DSupplyDrawModuleData, m_supplyBonePrefix) },
+        { nullptr, nullptr, nullptr, 0 },
+    };
+    // clang-format on
+
+    W3DModelDrawModuleData::Build_Field_Parse(p);
+    p.Add(dataFieldParse, 0);
+}
+
+NameKeyType W3DSupplyDraw::Get_Module_Name_Key() const
+{
+    static const NameKeyType nk = g_theNameKeyGenerator->Name_To_Key("W3DSupplyDraw");
+    return nk;
+}
+
+void W3DSupplyDraw::Update_Draw_Module_Supply_Status(int max, int current)
+{
+    W3DModelDraw::Update_Draw_Module_Supply_Status(max, current);
+    Utf8String prefix = Get_W3D_Supply_Draw_Module_Data()->m_supplyBonePrefix;
```

Comment:
No need to "copy" here. `const Utf8String &prefix` will suffice.

Reply:
fixed.